### DISCUSSION TITLE
setBrightScreen() overrides "Screen Always Lit" setting?

### DISCUSSION THREAD

**Discussion by melianmiko** [zepp-health/zeppos-forum#195] [Developer Feedback]
Hi,

Received a bugreport from one Amazfit Band 7 (ZeppOS 1.0) user about that behaviour. Steps to reproduce:

1. Enable "Screen always lit" in device settings for any time (5 minutes, for example);
2. Launch any application that uses `hmSetting.setBrightScreen()` and `hmSetting.setBrightScreenCancel()` (for example, my Tasks app from store);
3. Leave application and check "Screen always lit" option status — it will be disabled and screen will dim after 5 seconds.

How I can bypass this behaviour? 
Does it's a way to get/set "Screen always lit" option value?

**Comment by MyoungXUE** [comment]
@melianmiko Hi, thank you very much for your feedback, I have reproduced the problem.

There is no workaround in Zepp OS 1.0 as of now, the setBrightScreen API overrides the bright screen settings.

In subsequent releases we will optimize the bright screen API design to ensure that the bright screen behavior is as expected.
